### PR TITLE
docs: Updates system setup

### LINKED ISSUE #4060: docs: write a system setup guide for linux
Write a system setup guide for Linux (see https://invenio.readthedocs.io/en/master/getting-started/development-environment.html#linux).Use the macOS example on the same page as a template.

I would think we would likely need to split the guide into 
- Ubuntu/Debian
- CentOS

Create a VM with the system (CentOS or Ubuntu/Debian) to test with so that we don't forget important things for newcomers.

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes #4060

### CODE REVIEW

File: docs/getting-started/development-environment.rst [hunk 1]
```diff
@@ -58,6 +64,22 @@ using the command:
 
     $ brew install <packages>
 
+If you are on linux you can use the native package manager instead of Homebrew.
```
Comment (lnielsen) [inline docs/getting-started/development-environment.rst:67]: Question: Is the package names not different between brew, yum and apt?

**Review by lnielsen — COMMENTED** [review]
Minor: The intention was to have separate guides for macOS/Linux so that the guides are super easy to follow. Also, the Linux guide should include getting the developer tools like Discord, VS Code...

**Comment by jrcastro2** [comment]
Updated: Created a guide for Ubuntu

**Comment by ines-cruz** [comment]
closing in favor of #https://github.com/inveniosoftware/invenio/pull/4084

**Pull request closed without merging**